PORTAL/tests/data/jobs.json

```diff
@@ -0,0 +1,8 @@
+{
+    "local_user": "benchmarking",
```

Comment:
We'd probably need to adjust "local_user" to whoever is running the tests, meaning we'd likely need to generate the config file (at least the first time).

Reply:
Also, the config doesn't have "data_dir" set, so commands that use it will fall back to the default (our actual production data).  Instead, consider using a data dir rooted under tests/data.  It may need to be different depending on the test.

Reply:
Both of those could probably wait, but it would be worth opening an issue at least.

Reply:
Sorry -- I misspoke -- I did the dynamic `data_dir` thing in PR #108.  Let me move that over here.